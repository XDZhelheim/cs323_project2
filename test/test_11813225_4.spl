int this_is_test()
{
  int xx;
  if (a>b)
  {
    return -1;
  }
  if (c==d)
  {
    return 1.1;
  }
}