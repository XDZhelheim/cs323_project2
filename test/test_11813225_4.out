Error type 1 at Line 4: a variable is used without a definition
Error type 1 at Line 4: a variable is used without a definition
Error type 1 at Line 8: a variable is used without a definition
Error type 1 at Line 8: a variable is used without a definition
Error type 8 at Line 10: a functionâ€™s return value type mismatches the declared type
